readme.txt

```diff
@@ -216,6 +216,12 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 == Changelog ==
 
+= [4.7.4] TBD =
+
+* Fix - Display the tags block delimiter in the editor [119393]
+
+= [4.7.3] TBD =
```

Comment:
I think this one should be 18.19 as `18.18` is `4.7.2`

Reply:
Thanks @mitogh! I've been asked to work directly on 19.01 so we can leave a clean Q for 18.19

Reply:
OMG! Didn't realize about that, so welcome to the first commit for the 2019 MR.

Reply:
🙌